WPFHospitalEditor/AllMapObjects.cs

```diff
@@ -12,9 +16,15 @@ class AllMapObjects
         public static List<MapObject> allFirstBuildingSecondFloorObjects = new List<MapObject>();
         public static List<MapObject> allSecondBuildingFirstFloorObjects = new List<MapObject>();
         public static List<MapObject> allSecondBuildingSecondFloorObjects = new List<MapObject>();
+        public static List<MapObject> allMapObjects = new List<MapObject>();
 
+        
+        FileRepository path1 = new FileRepository(AllConstants.MAPOBJECT_PATH);
```

Comment:
unnecessary line

Reply:
Path is now declared in MapObjectRepository, and FileRepository class is deleted, so this line is deleted as well.